components/Dialogs/src/private/MDCAlertControllerView+Private.m

```diff
@@ -230,6 +230,17 @@ - (void)updateButtonFont {
   }
   for (MDCFlatButton *button in self.actionButtons) {
     [button setTitleFont:finalButtonFont forState:UIControlStateNormal];
+    [button sizeToFit];
+    CGRect buttonRect = button.frame;
+    buttonRect.size.height = MAX(CGRectGetHeight(buttonRect), MDCDialogActionButtonHeight);
+    buttonRect.size.width = MAX(CGRectGetWidth(buttonRect), MDCDialogActionButtonMinimumWidth);
+    button.frame = buttonRect;
+    CGFloat verticalInset = MIN(0, -(48 - CGRectGetHeight(button.bounds)) / 2);
+    CGFloat horizontalInset = MIN(0, -(48 - CGRectGetWidth(button.bounds)) / 2);
+    button.hitAreaInsets = UIEdgeInsetsMake(verticalInset,
+                                                  horizontalInset,
+                                                  verticalInset,
+                                                  horizontalInset);
```

Comment:
If we are resizing view, we likely want to be dealing that code in layoutSubviews.  Generally, not just in this specific case.

At the top of layoutSubviews, where we are calling [button sizeToFit] on all the actions, I would add this code there.